### PR TITLE
Use data-task library

### PR DESCRIPTION
**Pull request by marcospri**
For:

 * https://github.com/hypothesis/report/issues/47

## Testing

-  `tox -e dev --run-command "python bin/run_data_task.py --task report/create_from_scratch"`

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -4,6 +4,8 @@
 #
 #    pip-compile --allow-unsafe requirements/dev.in
 #
+-e file:///home/marcos/hypo/data-tasks
```
Comment (marcospri) [inline requirements/dev.txt:7]: Temporary location until publushing the library

File: .cookiecutter/includes/requirements/prod.in [hunk 2]
```diff
@@ -3,9 +3,6 @@ newrelic
 
 hubspot-api-client
 importlib_resources
-jinja2
```
Comment (marcospri) [inline .cookiecutter/includes/requirements/prod.in:6]: These are now transitive dependencies.

File: report/db.py [hunk 3]
```diff
@@ -4,6 +4,6 @@
 SESSION = sessionmaker()
 
 
-def make_engine(settings):
+def make_engine(settings):  # pragma: no cover
```
Comment (marcospri) [inline report/db.py:7]: We only use this in the functest.

We could adapt it there but I reckon we'd bring it back at some point and forget about the other one.

File: tests/unit/conftest.py [hunk 4]
```diff
@@ -1,40 +0,0 @@
-import pytest
-import sqlalchemy
-
-from report.db import SESSION
-
-
-@pytest.fixture
-def db_session(db_engine):
```
Comment (marcospri) [inline tests/unit/conftest.py:8]: We don't use this one at all, if we need it again we can just copied it over again.

File: Makefile [hunk 5]
```diff
@@ -25,6 +25,11 @@ $(call help,make shell,"launch a Python shell in this project's virtualenv")
 shell: python
 	@pyenv exec tox -qe dev --run-command 'ipython'
 
+.PHONY: sql
```
Comment (marcospri) [inline Makefile:28]: Unrelated but with this `make template` is now clean.

**Review by jon-betts — APPROVED** [review]
This all looks good, and it's a big chunk of code to consolidate when all is said and done.

One thing I think we should follow up on is the `sql_tasks/tasks` directory is now redundant, as there's no code to separate from the tasks. I think we can / should bump that directory up one.

This will also make it slightly less irritating to work with as it's one less directory to step into when fixing symlinks etc.

**Pull request merged**